docs/upgrading/splinter-v0.3.15-from-v0.3.14.md

```diff
@@ -0,0 +1,8 @@
+# Upgrading to Splinter v0.3.15 from Splinter v0.3.14
+
+Breaking changes and significant changes in this release
+
+* The routes for `biome/users/{user_id}/keys` and
```

Comment:
This should probably be stated in two pieces. 1) {user_id} has been removed from all 'biome/users/ routes. (w/example paths) and 2) keys have been moved from biome/users/ to biome/keys/ (with examples).

Reply:
We still have a `biome/users/{id}` route, so 1) wouldn't be accurate, unless you're suggesting we remove id from those routes as well. I initially left those alone, because I wasn't sure if having someone other than the logged in user access another user's user id was a valid user case.